docs/source/development.md

```diff
@@ -29,15 +29,25 @@ docker build \
 Run this inside the `pipeline/` folder.
 
 ```
+pip3 install -r requirements.txt
 DB_USER=$DB_USER \
 DB_PASSWORD=$DB_PASSWORD \
 DB_HOST=$DB_HOST \
 DB_PORT=$DB_PORT \
 DB_DATABASE=$DB_DATABASE \
 GH_TOKEN=$GH_TOKEN \
-python3 -m unittest -v
+coverage3 run --source scraper,model --omit */__init__.py,*/tests/* -m unittest
```

Comment:
zsh doesn't like this (bash is fine, which is why it works for you and @vpopil). I found a workaround though, just quote the argument to `--omit` and it will work on both shells.

```suggestion
coverage3 run --source scraper,model --omit "*/__init__.py,*/tests/*" -m unittest
```

Reply:
Ooh, ok. Yea, it works on my end as well. Will accept not. Thanks for catching that